shared/components/modals/OfferModal/AddOffer/AddOffer.js

```diff
@@ -364,10 +371,37 @@ export default class AddOffer extends Component {
     })
   }
 
+  chooseProps = () => {
+    const { currencies, addSelectedItems } = this.props
 
+    if (addSelectedItems === undefined) {
+      return currencies
+    }
+    return addSelectedItems
+  }
+
+  zeroPosition = () => {
+    const { addSelectedItems } = this.props
+    const { buyCurrency, seellCurrency } = this.state
```

Comment:
```suggestion
     const { buyCurrency, sellCurrency } = this.state
```

почему тут опечатка?